app/data/AtomListStore.scala

```diff
@@ -103,7 +102,7 @@ class DynamoBackedAtomListStore(store: AtomListStore.PreviewStore) extends AtomL
 }
 
 object AtomListStore {
-  type PreviewStore = PreviewDynamoDataStore[ThriftMediaAtom]
+  type PreviewStore = PreviewDynamoDataStore
```

Comment:
Do you actually use this in many places? It might not be worth defining a new type for something this basic now.